neo/Plugins/Plugin.cs

```diff
@@ -95,7 +97,10 @@ internal static void LoadPlugins(NeoSystem system)
                     {
                         constructor?.Invoke(null);
                     }
-                    catch { }
+                    catch (Exception ex)
+                    {
+                        Console.WriteLine("Failed to initialize plugin: " + ex); 
```

Comment:
```suggestion
                        Plugin.Log(nameof(Plugin), LogLevel.Error, $"Failed to initialize plugin: {ex.Message}"); 
```

Reply:
Implemented suggested change in both exception handlers https://github.com/neo-project/neo/pull/521/commits/b6e4a999fe232d4f2b3cf86e7a814ff789bdc7e0